spec/integration/rack_spec.rb

```diff
@@ -49,6 +49,8 @@
 
   let(:s3_double) { instance_double("Alephant::Storage", :get => content) }
 
+  let(:not_modified_status_code) { 202 }
```

Comment:
You could, rather than hard coding the status code here do `described_class::NOT_MODIFIED_STATUS_CODE`